src/plugins/vis_types/vega/public/vega_view/vega_tooltip.js

```diff
@@ -79,12 +79,19 @@ export class TooltipHandler {
       anchorBounds = createRect(event.clientX, event.clientY, 0, 0);
     } else {
       const containerBox = this.container.getBoundingClientRect();
-      anchorBounds = createRect(
-        containerBox.left + view._origin[0] + item.bounds.x1,
-        containerBox.top + view._origin[1] + item.bounds.y1,
-        item.bounds.width(),
-        item.bounds.height()
-      );
+      let left = containerBox.left + view._origin[0] + item.bounds.x1;
+      let top = containerBox.top + view._origin[1] + item.bounds.y1;
+
+      // loop item mark group
+      let ancestorItem = item;
+
+      while (ancestorItem.mark.group) {
+        ancestorItem = ancestorItem.mark.group;
+        left += ancestorItem.x;
+        top += ancestorItem.y;
+      }
+
+      anchorBounds = createRect(left, top, item.bounds.width(), item.bounds.height());
```

Comment:
These changes look okay to me. @dej611 or @markov00 do you see any issues with this?